src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void register_incorrectAge_NotOk() {
+        User user = new User(null, "qwertyd", "1234567", 17);
+        assertThrows(InvalidDataException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_incorrectPassword_NotOk() {
```

Comment:
method name lacks clarity of the password, lets name add more information on what is incorrect about this password
```suggestion
    void register_shortPassword_NotOk() {
```